upup/models/cloudup/resources/addons/networking.amazon-vpc-routed-eni/k8s-1.16.yaml.template

```diff
@@ -106,42 +106,35 @@
                 - "fargate"
       "containers":
       - "env":
-        - "name": "ADDITIONAL_ENI_TAGS"
-          "value": "{}"
-        - "name": "AWS_VPC_CNI_NODE_PORT_SUPPORT"
-          "value": "true"
-        - "name": "AWS_VPC_ENI_MTU"
-          "value": "9001"
         - "name": "AWS_VPC_K8S_CNI_CONFIGURE_RPFILTER"
-          "value": "false"
-        - "name": "AWS_VPC_K8S_CNI_CUSTOM_NETWORK_CFG"
-          "value": "false"
-        - "name": "AWS_VPC_K8S_CNI_EXTERNALSNAT"
-          "value": "false"
-        - "name": "AWS_VPC_K8S_CNI_LOGLEVEL"
-          "value": "DEBUG"
-        - "name": "AWS_VPC_K8S_CNI_LOG_FILE"
-          "value": "/host/var/log/aws-routed-eni/ipamd.log"
-        - "name": "AWS_VPC_K8S_CNI_RANDOMIZESNAT"
-          "value": "prng"
-        - "name": "AWS_VPC_K8S_CNI_VETHPREFIX"
-          "value": "eni"
-        - "name": "AWS_VPC_K8S_PLUGIN_LOG_FILE"
-          "value": "/var/log/aws-routed-eni/plugin.log"
-        - "name": "AWS_VPC_K8S_PLUGIN_LOG_LEVEL"
-          "value": "DEBUG"
-        - "name": "DISABLE_INTROSPECTION"
-          "value": "false"
-        - "name": "DISABLE_METRICS"
-          "value": "false"
-        - "name": "ENABLE_POD_ENI"
-          "value": "false"
+          "value": "{{ AwsVpcK8sCniConfigureRpFilter }}"
+        # The below envs are commented-out on purpose. See https://github.com/kubernetes/kops/issues/11144 for more context.
+        # - "name": "AWS_VPC_K8S_CNI_CUSTOM_NETWORK_CFG"
+        #   "value": "false"
+        # - "name": "AWS_VPC_K8S_CNI_EXTERNALSNAT"
+        #   "value": "false"
+        # - "name": "AWS_VPC_K8S_CNI_LOGLEVEL"
+        #   "value": "DEBUG"
+        # - "name": "AWS_VPC_K8S_CNI_LOG_FILE"
+        #   "value": "/host/var/log/aws-routed-eni/ipamd.log"
+        # - "name": "AWS_VPC_K8S_CNI_RANDOMIZESNAT"
+        #   "value": "prng"
+        # - "name": "AWS_VPC_K8S_CNI_VETHPREFIX"
+        #   "value": "eni"
+        # - "name": "AWS_VPC_K8S_PLUGIN_LOG_FILE"
+        #   "value": "/var/log/aws-routed-eni/plugin.log"
+        # - "name": "AWS_VPC_K8S_PLUGIN_LOG_LEVEL"
+        #   "value": "DEBUG"
+        # - "name": "DISABLE_INTROSPECTION"
+        #   "value": "false"
+        # - "name": "DISABLE_METRICS"
+        #   "value": "false"
+        # - "name": "ENABLE_POD_ENI"
+        #   "value": "false"
         - "name": "MY_NODE_NAME"
           "valueFrom":
             "fieldRef":
               "fieldPath": "spec.nodeName"
-        - "name": "WARM_ENI_TARGET"
-          "value": "1"
```

Comment:
Did we want to comment this one out as well?

Reply:
Done 👍🏻 
Good catch, thanks!